### PR TITLE
tag table updates when merge button clicked

### PR DESCRIPTION
**Pull request by jackgrossman18**
ref #1480

### CODE REVIEW

**Comment by jackgrossman18** [comment]
@brianhatchl @maxgrossman this is ready for review

**Comment by jackgrossman18** [comment]
cc @maxgrossman

File: modules/Hoot/ui/conflicts/conflicts.js [hunk 1]
```diff
@@ -149,6 +178,39 @@ export default class Conflicts {
         this.bindKeys();
     }
 
+    /**
+     * Update buttons
+     */
+
+     updateButtons() {
```
Comment (brianhatchl) [inline modules/Hoot/ui/conflicts/conflicts.js]: couldn't this be used by the create above?

**Comment by brianhatchl** [comment]
I also noticed that if you use <kbd>Ctrl</kbd> to reverse the merge direction, the red column shows when it should be blue.

**Comment by jackgrossman18** [comment]
@brianhatchl this is ready for review

**Comment by brianhatchl** [comment]
I think a bad merge happened because there are a bunch of good upstream changes being removed in this PR.

**Pull request closed without merging**